packages/app-storage/src/Query.tsx

```diff
@@ -37,6 +38,62 @@ type CacheInstance = {
 
 const cache: Array<CacheInstance> = [];
 
+enum StorageQueryParameter {
```

Comment:
Why not the actual types enum - here we now, when changing it in types, remember to change things here. Fragile :(